src/Utility/Hash.php

```diff
@@ -43,7 +44,7 @@ class Hash
      * @return mixed The value fetched from the array, or null.
      * @link http://book.cakephp.org/3.0/en/core-libraries/hash.html#Hash::get
      */
-    public static function get(array $data, $path, $default = null)
+    public static function get($data, $path, $default = null)
```

Comment:
Nothing good I guess :)  Should I add a type check and throw exception if not an array or ArrayAccess instance?